editor-meta.Rmd

```diff
@@ -0,0 +1,26 @@
+# Editorial governance {#editormeta}
+
+## GitHub team
+
+rOpenSci blog editors are the member of the [`@ropensci/blog-editors`](https://github.com/orgs/ropensci/teams/blog-editors) team.
+
+The team is maintained by [`@stefaniebutland`](https://github.com/stefaniebutland) at the moment.
+
+* Blog editors can be tagged in [any ropensci repo](https://github.com/ropensci/), in particular [roweb2](https://github.com/ropensci/roweb2/), by writing `@ropensci/blog-editors` (e.g. `:wave: @ropensci/blog-editors, I've an issue with using DT in my post!`).
+
+* Blog editors have write access to the roweb2 repo.
+
+* Blog editors are ["code owners"](https://github.com/ropensci/roweb2/blob/master/.github/CODEOWNERS) of `roweb2/content/technotes`, `roweb2/content/blog`, `roweb2/content/commcalls` according to [roweb2's CODEOWNERS file ](https://github.com/ropensci/roweb2/blob/master/.github/CODEOWNERS) so any PR touching a file under these paths will automatically request their review. The blog editor that will effectively handle the PR should unrequest the review from other blog editor(s).
+
+For adding a new blog editor, the team maintainer or any ropensci admin
+
+* ensures the user has [enabled 2FA](https://help.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa),
+
+* [adds them to the team](https://github.com/orgs/ropensci/teams/blog-editors/members) as Member (the default role).
+
+BLABLABLA what to add regarding access to this very repo i.e. blog-guidance?
+
+## Posts authored by staff
```

Comment:
Maybe add something about staff members' being responsible for

* tweeting about their post from rOpenSci Twitter account (either direct access or Tweetdeck access)
* creating the discuss forum blabla

In both cases add a link to the relevant sections of the editor guide?

Reply:
Also, do staff members have to tweet at 8am Pacific or can they tweet at any time?

Reply:
I will add this before merging (I *think* I can commit, then merge. Can't recall if I tried before)

Reply:
committed